### ISSUE TITLE
TT-Topo Initialization Error

### ISSUE THREAD

**Issue by hmohiuddinTT** [tenstorrent/tt-topology#1]
Running into an issue running tt-topo. Looks like there was a change to the common reset package. @sbansalTT taking a look.

![image](https://github.com/tenstorrent/tt-topology/assets/124714595/1f532cae-e82d-4fc5-bfcf-6c321af7f0a5)

**Comment by jliangTT** [comment]
@tapspatel , did you see this issue on your side before?

**Comment by hmohiuddinTT** [comment]
@jliangTT this seems like an error due to a recent change in one of the dependencies of this script. @sbansalTT has already pushed a fix, just waiting for it to propagate to the public GitHub repo.

**Comment by sbansalTT** [comment]
Ok fixed as of v1.0.2!

```
$~/tt-topology > tt-topology --help
usage: tt-topology [-h] [-v] [-l {linear,torus,mesh}] [-ls] [--log [log]] [-p [plot]]

Tenstorrent Topology (TT-Topology) is a command line utility to flash ethernet coordinates when multiple NB's are connected together.

optional arguments:
  -h, --help            show this help message and exit
  -v, --version         show program's version number and exit
  -l {linear,torus,mesh}, --layout {linear,torus,mesh}
                        Select the layout (linear, torus, mesh). Default is linear.
  -ls, --list           List out all the boards on host with their coordinates and layout.
  --log [log]           Change filename for the topology flash log. Default: ~/tt_topology_logs/<timestamp>_log.json
  -p [plot], --plot_filename [plot]
                        Change the plot of the png that will have the graph layout of the chips. Default: chip_layout.png
```
